src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -449,7 +454,7 @@ static irep_idt get_any_incomplete_ancestor(
       classes_to_check.end(), parents.begin(), parents.end());
   }
 
-  INVARIANT(false, "input class id should have some incomplete ancestor");
+  return irep_idt();
```

Comment:
Can we add a TODO here so that we come back and fix it.

Reply:
Done at the callsite